app/models/episode.rb

```diff
@@ -83,7 +110,9 @@ def read_id3
         self.length = self.length || mp3info.tag2.TLEN
         self.summary = self.summary || mp3info.tag2.COMM
 
-        #TODO image = mp3info.tag2.APIC ftp?
+        if mp3info.tag2.pictures.first
+          self.image = self.image || StringIO.new(mp3info.tag2.pictures.first.last)
```

Comment:
self.image is never nil so this should check against "/images/original/missing.png"